app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -168,13 +168,19 @@ function initPagination() {
               if (markup) {
                 const container = document.getElementById('articles-list');
 
+                const scrollTop =
+                  window.pageYOffset || document.documentElement.scrollTop;
+
                 const newNotifications = document.createElement('div');
                 newNotifications.innerHTML = markup;
 
                 paginator.remove();
                 container.append(newNotifications);
 
                 initReactions();
+
+                // Restoring scroll position
+                document.documentElement.scrollTop = scrollTop;
```

Comment:
I'm actually saving the scroll position and restoring it after loading more elements. Is this ok ?